### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/blog/blogger.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#699] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/blog/blogger.html

主题允许你展示博主的基本信息。

https://theme-hope.vuejs.press/zh/guide/blog/blogger.html

**Comment by wadeKings** [comment]
请问author.url是干啥的

**Reply by Mister-Hope** [reply]
这页面里也没提到这个东西啊？

**Reply by HK-bruc1** [reply]
弱弱的同问，没填写blog.avatar 和 blog.name。它们会自动回退到站点 Logo (主题选项中的 logo) 和 站点名称。而name会回退到author配置里。但是url我也没明白是干嘛的
author: {
    name: "balabala",
    url: "https://mister-hope.com",  
  },

**Reply by Mister-Hope** [reply]
作者的个人网站

**Comment by dragon6long** [comment]
所以blog{ } 里的参数为什么没有上面提到的

**Comment by Alex6357** [comment]
SVG路径必须要http开头的路径吗？放在public下的文件不可以写绝对路径？
